scripts/no-internal-javadoc.gradle

```diff
@@ -34,6 +43,10 @@ task noInternalJavadoc(type: Javadoc) {
         !it.absolutePath.contains('generated')
     }
     classpath = javadoc.getClasspath()
-    options.doclet = 'org.spine3.tools.javadoc.ExcludeInternalDoclet'
-    options.docletpath = configurations.jaxDoclet.files.asType(List)
+
+    options {
```

Comment:
Please have a comment for this section (or for the items of the section separately).